src/SonarQube.SonarLint.Runner/Program.cs

```
@@ -34,12 +33,27 @@ public static class Program
     {
         public static int Main(string[] args)
         {
+            if (args.Length != 3)
+            {
+                Write("Expected parameters: ");
+                Write("[Input configuration path: should end with sonarlint.xml, otherwise rule parameter value will be ignored]");
```

@tvsonar the ": should end with sonarlint.xml, otherwise rule parameter value will be ignored]" part can be removed - the fail fast behavior will make it explicit enough.